AlphaWallet/GoogleService-Info.plist

```diff
@@ -2,6 +2,8 @@
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
 <dict>
+	<key>FirebaseCrashlyticsCollectionEnabled</key>
```

Comment:
Does this file have to be updated? Because I keep a copy of the prod version for submission.

Reply:
yes, i guess we don't need to upload it

Reply:
in case for tests, what file do we need to use?

Reply:
For tests locally and Travis? I think the one you added `AlphaWallet/GoogleService-InfoTests.plist` looks good. Was just wondering why you updated `AlphaWallet/GoogleService-Info.plist` too (because then it might mean I need to update the prod copy)

Reply:
oh i see, updated it with manually setting FirebaseCrashlytics Collection Enabled, to avoid misunderstanding.